src/AppBundle/Twig/AppExtension.php

```diff
@@ -49,9 +49,9 @@ public function markdownToHtml($content)
     }
 
     // the name of the Twig extension must be unique in the application. Consider
-    // using 'app.extension' if you only have one Twig extension in your application.
+    // using 'app_extension' if you only have one Twig extension in your application.
```

Comment:
What if we had `UserType` form type and `UserExtension` twig extension in app bundle? Then we had a collision with `app_user` if we remove `_extension` suffix. So I don't like idea to remove `_extension` suffix. However, form types use much often then Twig extensions, so they had exclusion from `_type` suffix. I'll be better to add `_type` suffix to form type than remove `_extension` one.